tests/lib/errors/ExpectedValidationError.test.ts

```diff
@@ -0,0 +1,41 @@
+import { inspect } from 'node:util';
+import { ExpectedValidationError } from '../../../src';
+
+describe('ExpectedValidationError', () => {
+	const error = new ExpectedValidationError('LiteralValidator', 'Expected values to be equals', 'world', 'hello');
+
+	test('GIVEN an instance THEN assigns fields correctly', () => {
+		expect(error.message).toBe('Expected values to be equals');
+		expect(error.validator).toBe('LiteralValidator');
+		expect(error.given).toBe('world');
+		expect(error.expected).toBe('hello');
+	});
+
+	describe('inspect', () => {
+		test('GIVEN an inspected instance THEN formats data correctly', () => {
+			const content = inspect(error, { colors: false });
+			const expected = [
+				'ExpectedValidationError > LiteralValidator', //
+				'  Expected values to be equals',
+				'',
+				'  Expected:',
+				"  | 'hello'",
+				'',
+				'  Received:',
+				"  | 'world'",
+				''
+			];
+
+			expect(content.startsWith(expected.join('\n'))).toBe(true);
+		});
+
+		test('GIVEN an inspected instance with negative depth THEN formats name only', () => {
+			const content = inspect(error, { colors: false, depth: -1 });
+			const expected = [
+				'[ExpectedValidationError]' //
```

Comment:
at least always include the subsequent error name (`[ExpectedValidationError > LiteralValidator]` or just `[LiteralValidator]`